### ISSUE TITLE
Support APNG

### ISSUE THREAD

**Issue by CasperHK** [phw/peek#108]
Will peer support APNG in the future version?

**Comment by cmedya** [comment]
@CasperHK i agree with you.that's more like it .

**Comment by phw** [comment]
I thought about this, too, when reading that Chrome 59 will add support APNG. Not 100% sure yet, as this would require different tools (both imagemagick and ffmpeg do not handle APNG). Have to investigate a bit more. But I actually would like to support it, as it fits well into what Peek is doing.

Also see my comment on https://github.com/phw/peek/issues/89#issuecomment-283091038 for some more information on APNG and WebP. But the point about browser support will change if Chrome jumps in and support it. For MS Edge it is at least under review: https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/6513393-apng-animated-png-images-support-firefox-and-sa

**Comment by phw** [comment]
I just learned that ffmpeg can generate apng. Not quite sure since whenthis was available, the official documentation also does not mention the APNG muxer, but it exists (https://trac.ffmpeg.org/wiki/SponsoringPrograms/GSoC/2015#AnimatedPortableNetworkGraphicsAPNG). And with my current changes to use ffmpeg also for GIF this becomes quite trivial to add, so expect this in the next release :D

Same is true for WebP by the way, although I tried it and was not happy with the quality produced.
